response.go

```diff
@@ -70,19 +81,19 @@ func (err connectionError) Timeout() bool {
 	return true
 }
 
-// serverError represents internal server errors.
+// ServerError represents internal server errors.
 // Implements `net.Error` interface.
-type serverError string
+type ServerError string
```

Comment:
So, I will merge this PR and then, maybe I will change error exporting. Thank you for this PR!